src/test/Components/CreateImageWizard/steps/ImageOutput/ImageOutput.test.tsx

```diff
@@ -127,9 +127,12 @@ const selectVMwareTarget = async () => {
   await waitFor(() => user.click(vmwareImageCheckBox));
 };
 
-const goToReviewStep = async () => {
+const handleRegistration = async () => {
```

Comment:
Splitting registration and skipping between steps into two so we can just skip when CentOS selected.